Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_5.md

```diff
@@ -0,0 +1,16 @@
+
+#### Playbooks
+
+##### Cortex ASM - Remediation
+
+Added the new inputs for *Cortex ASM - On Prem Remediation* playbook.
+
+##### Cortex ASM - On Prem Enrichment
+
+Playbook now supports collecting information on offending rules in Panorama.
+
+##### Cortex ASM - On Prem Remediation
+
+Added the following inputs to support blocking internet exposures via Panorama:
+ - *DeviceGroup* - Device Group of the firewall rule to lookup
+ - *SecondaryDeviceGroup* - In that case that the rule, address and service are created in the "Shared" location, we need to know what device-groups we can push to because it isn't possible to push to the "Shared" location.
```

Comment:
```suggestion
 - *SecondaryDeviceGroup* - If the rule, address and service are created in the "Shared" location, we need to know what device-groups we can push to because it isn't possible to push to the "Shared" location.
```